src/main/java/com/ldtteam/structurize/generation/StructurizeTranslations.java

```diff
@@ -0,0 +1,133 @@
+package com.ldtteam.structurize.generation;
+
+import com.ldtteam.structurize.api.generation.ModLanguageProvider;
+import com.ldtteam.structurize.blocks.ModBlocks;
+import com.ldtteam.structurize.items.ModItemGroups;
+import com.ldtteam.structurize.items.ModItems;
+import net.minecraft.data.DataGenerator;
+
+public class StructurizeTranslations extends ModLanguageProvider
+{
+    public StructurizeTranslations(final DataGenerator gen, final String modid, final String locale)
+    {
+        super(gen, modid, locale);
+    }
+
+    @Override
+    protected void addTranslations()
+    {
+        super.addTranslations();
+
+        add(ModBlocks.multiBlock.get(), "Multi-Piston");
+        add(ModBlocks.blockDecoBarrel_onside.get(), "Barrel Horizontal");
+        add(ModBlocks.blockDecoBarrel_standing.get(), "Barrel Vertical");
+        add(ModBlocks.blockSubstitution.get(), "Placeholder Block");
+        add(ModBlocks.blockSolidSubstitution.get(), "Solid Placeholder");
+        add(ModBlocks.blockFluidSubstitution.get(), "Fluid Placeholder");
+        add(ModItems.caliper);
+        add(ModItems.buildTool.get(), "Build Tool");
+        add(ModItems.scanTool.get(), "Scan Tool");
+        add(ModItems.tagTool.get(), "Tag Tool");
+        add(ModItems.shapeTool.get(), "Shape Tool");
+        add(ModItemGroups.STRUCTURIZE, "Structurize");
+        add(ModItemGroups.CONSTRUCTION, "Structurize Construction Blocks");
+        add(ModItemGroups.SHINGLES, "Structurize Shingles");
+        add(ModItemGroups.TIMBER_FRAMES, "Structurize Timber Frames");
```

Comment:
Agreed

Reply:
This was me following direction from Asher in a brief discussion we had.
It was a toss-up between this option, and having the provider read and copy the manual one to the generated one.
Between these two options, there should be no reason to keep the manual and genned keys in the same file. 

For the record I preferred the latter one, but this one is growing on me, for the following advantages:
 - Using these add overloads it is impossible to accidentally misspell a block or item key
 - You can use autocomplete in your IDE for the variable based overloads
 - The order in any lang file doesn't really matter, as it can be determined here.
 - Allows easy access to the provider should anything dynamic be needed that isn't covered by the collection/list interfaces
 - Once more overloads are added (for 1.17) it will begin to enforce consistency. 
 - Done correctly this will *increase* maintainability, or effectively remain the same.
 
I haven't looked deeply into CrowdIn, but if you can't just switch the input to the generated file I'd be very surprised. No need to mess with code at all.

Reply:
- regarding IDE and overloads, you can still fail doesn't matter (guis/commands etc.), we will just add more overloads over time making it even less maintainable, also - the biggest drawbacks - reloads, you can't reload or it's super annoying to reload genned files during debug

- the order does matter for both, genned needs to be sorted cuz of git, manual is usually "foldered" by usage type etc.

- crowdin is irrelevant in this problem (but the problems above still applies), you can merge jsons in gradle

Reply:
- The overloads are one-liners, I would hardly consider that to be a maintainability issue until you get to 20 or so of them. You make a good point about reload though - reloading/regenning time was why this PR took so long!
- I guess what I was trying to get at was the "foldered" style is a bit more visually apparent due to syntax highlighting. But its a subtle advantage, and depends how you look at it I guess.
- I did not know gradle could do that. And the multiple jsons was what I asked in Discord that started this! So if you know how to do it I'm quite happy to use that method. I'll just blame Asher.

Reply:
> I did not know gradle could do that. And the multiple jsons was what I asked in Discord that started this! So if you know how to do it I'm quite happy to use that method. I'll just blame Asher.

google it :D gradle is "basically" another semi-java code so you can just import json classes and do whatever you want

Reply:
remove this file?